ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -182,7 +230,9 @@ class PropertySpec: QuickSpec {
 					let property = AnyProperty(constantProperty)
 
 					var sentValue: String?
+					var signalSentValue: String?
 					var producerCompleted = false
+					var signalInterrupted = false
```

Comment:
External observers can never catch a `Completed` event from the signal of a `ConstantProperty`, and doing `signal.observe` on such a signal would result in an `Interrupted` event passed to the `Observer`.